src/generator.cs

```diff
@@ -5552,6 +5555,7 @@ void PrintExport (MemberInformation minfo, string sel, ArgumentSemantic semantic
 
 	void GenerateMethod (MemberInformation minfo)
 	{
+		using var span = GeneratorContext.BeginSpan ($"Method: {minfo.method.Name}");
```

Comment:
The generator is already slow, and this will create a new string every time it's hit. Not sure how to improve it though.